src/ontology/catalog-v001.xml

```diff
@@ -1,60 +1,58 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
     <group id="Folder Repository, directory=, recursive=false, Auto-Update=false, version=2" prefer="public" xml:base="">
-      <uri name="http://purl.obolibrary.org/obo/uberon.owl" uri="tmp/ext-weak.owl"/>
-      <uri name="http://purl.obolibrary.org/obo/fma.owl" uri="components/null.owl"/>
-      <uri name="http://purl.obolibrary.org/obo/uberon/ssso-merged-uberon.owl" uri="tmp/developmental-stage-ontologies/src/ssso-merged.obo"/>
-      <uri name="http://purl.obolibrary.org/obo/uberon/bridge/collected-vertebrate.owl" uri="bridge/collected-vertebrate.owl"/>
-      <uri name="http://purl.obolibrary.org/obo/uberon/bridge/collected-tetrapod.owl" uri="bridge/collected-tetrapod.owl"/>
-      <uri name="http://purl.obolibrary.org/obo/uberon/bridge/collected-teleost.owl" uri="bridge/collected-teleost.owl"/>
-      <uri name="http://purl.obolibrary.org/obo/uberon/bridge/collected-amniote.owl" uri="bridge/collected-amniote.owl"/>
-      <uri name="http://purl.obolibrary.org/obo/uberon/bridge/collected-mammal.owl" uri="bridge/collected-mammal.owl"/>
-      <uri name="http://purl.obolibrary.org/obo/uberon/bridge/collected-adult-mammal.owl" uri="bridge/collected-adult-mammal.owl"/>
-      <uri name="http://purl.obolibrary.org/obo/uberon/bridge/collected-embryonic-mammal.owl" uri="bridge/collected-embryonic-mammal.owl"/>
-      <uri name="http://purl.obolibrary.org/obo/mba.owl" uri="tmp/allen-mba.obo"/>
-      <uri name="http://purl.obolibrary.org/obo/pba.owl" uri="tmp/allen-pba.obo"/>
-      <uri name="http://purl.obolibrary.org/obo/hba.owl" uri="tmp/allen-hba.obo"/>
-      <uri name="http://purl.obolibrary.org/obo/dmba.owl" uri="tmp/allen-dmba.obo"/>
-      <uri name="http://purl.obolibrary.org/obo/dhba.owl" uri="tmp/allen-dhba.obo"/>
+      <uri name="http://purl.obolibrary.org/obo/ceph.owl" uri="imports/local-ceph.owl"/>
+      <uri name="http://purl.obolibrary.org/obo/cl.owl" uri="imports/local-cl.owl"/>
+      <uri name="http://purl.obolibrary.org/obo/cteno.owl" uri="imports/local-cteno.owl"/>
+      <uri name="http://purl.obolibrary.org/obo/dhba.owl" uri="imports/local-allen-dhba.obo"/>
```

Comment:
It’s OWL files for everything _except_ for the “Allen” ontologies, which are in OBO because they are custom-generated by [a script](https://github.com/obophenotype/uberon/blob/master/src/scripts/allen-json2obo.pl) that produces OBO files from whatever JSON we get from the Allen API. I have not changed anything about that.